src/argilla/client/feedback/training/schemas/base.py

```diff
@@ -1558,24 +1446,23 @@ def _format_data(self, dataset: "FeedbackDataset") -> List[Dict[str, Any]]:
             # NOTE: Maybe this post processing of the formatted data can be simplified
             # or directly done in super()._format_data(dataset).
             new_keys = {field.name: f"sentence-{i}" for i, field in enumerate(self.texts, start=1)}
-            if self.label:
+            if self.defaults.label:
                 new_keys.update({self.label.question.name: "label"})
 
             outputs = []
             for example in formatted_data:
                 record = {}
                 for k, v in new_keys.items():
+                    value = example[k]
                     if v == "label":
-                        value = example[v]
                         # At this point the label must be either an int or a float, determine which one is it.
-                        if isinstance(value, str):
-                            if value.lstrip("-").isdigit():
-                                value = int(value)
-                            else:
-                                value = float(value)
-                    else:
-                        value = example[k]
-
+                        if value.lstrip("-").isdigit():
+                            value = int(value)
+                        else:
+                            value = float(value)
+                        if isinstance(self.label, RankingQuestionUnification):
+                            max_value = max([float(x) for x in self.label.question.__all_labels__])
+                            value = (value / 100) * float(max_value)
```

Comment:
This was updated with https://github.com/argilla-io/argilla/pull/4171 to use the `RatingQuestionUnification`

Reply:
check

Reply:
check